### PR TITLE
retroarch: add page

### PR DESCRIPTION
**Pull request by kbdharun**
- [x] The page(s) are in the correct platform directories: `linux`.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

I added the GitHub repo link in `More information:`. We can replace it with [manned](https://manned.org/retroarch.1) if needed.

_Note: I had skipped some advanced commands like shaders, netplay, etc as there are a lot of examples and people rarely use it from the command line. I think basic commands are enough. Do I need to add more commands? What do you think?_

### CODE REVIEW

File: pages/linux/retroarch.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+> RetroArch is a frontend for emulators, game engines and media players.
+> It is the reference implementation of libretro API.
+> More information: <https://github.com/libretro/RetroArch>.
+
+- Start `retroarch` in menu mode:
+
+`retroarch`
+
+- Start `retroarch` in full screen mode:
```
Comment (EmilySeville7cfg) [inline pages/linux/retroarch.md:11]:
```suggestion
- Start `retroarch` in the full screen mode:
```

File: pages/linux/retroarch.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# retroarch
+
+> RetroArch is a frontend for emulators, game engines and media players.
+> It is the reference implementation of libretro API.
+> More information: <https://github.com/libretro/RetroArch>.
+
+- Start `retroarch` in menu mode:
```
Comment (EmilySeville7cfg) [inline pages/linux/retroarch.md:7]:
```suggestion
- Start `retroarch` in the menu mode:
```

Are there several menu modes? If so then prefer `a`.

File: pages/linux/retroarch.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+`retroarch --config={{path/to/config_file}}`
+
+- Display general help:
+
+`retroarch --help`
+
+- Display `retroarch` version:
+
+`retroarch --version`
```
Comment (EmilySeville7cfg) [inline pages/linux/retroarch.md:29]:
```suggestion
- Display the help:

`retroarch --help`

- Display the version:

`retroarch --version`
```

File: pages/linux/retroarch.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+- Start `retroarch` in menu mode:
+
+`retroarch`
+
+- Start `retroarch` in full screen mode:
+
+`retroarch --fullscreen`
+
+- List all features compiled into `retroarch`:
```
Comment (EmilySeville7cfg) [inline pages/linux/retroarch.md:15]:
```suggestion
- List all compiled features:
```

File: pages/linux/retroarch.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+`retroarch --features`
+
+- Set the path to configuration file:
```
Comment (EmilySeville7cfg) [inline pages/linux/retroarch.md:19]:
```suggestion
- Set the path of a configuration file:
```

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

File: pages/linux/retroarch.md (hunk 2 above)
Comment (kbdharun) [reply pages/linux/retroarch.md:7]: By default it goes into the menu, menu mode means it opens up the menu in retroarch. I can also open a game ROM directly without getting into menu mode. So, I will go with the current suggestion then.

**Review by EmilySeville7cfg — APPROVED** [review]
You can apply all review suggestions at once. 😄

File: pages/linux/retroarch.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+> RetroArch is a frontend for emulators, game engines and media players.
+> It is the reference implementation of libretro API.
+> More information: <https://github.com/libretro/RetroArch>.
+
+- Start in the menu mode:
+
+`retroarch`
+
+- Start in the full screen mode:
```
Comment (CleanMachine1) [inline pages/linux/retroarch.md:11]:
```suggestion
- Start in full screen mode:
```

File: pages/linux/retroarch.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+- List all compiled features:
+
+`retroarch --features`
+
+- Set the path of a configuration file:
+
+`retroarch --config={{path/to/config_file}}`
+
+- Display the help:
```
Comment (CleanMachine1) [inline pages/linux/retroarch.md:23]:
```suggestion
- Display help:
```

File: pages/linux/retroarch.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+
+`retroarch --help`
+
+- Display the version:
```
Comment (CleanMachine1) [inline pages/linux/retroarch.md:27]:
```suggestion
- Display version:
```

File: pages/linux/retroarch.md (hunk 7 above)
Comment (EmilySeville7cfg) [reply pages/linux/retroarch.md:23]: I don't think it's grammatically more correct without any article. 🤔

Comment (kbdharun) [reply pages/linux/retroarch.md]: > I don't think it's grammatically more correct without any article. thinking

I guess both will work. In this case, I think using without article would be fine.

**Review by CleanMachine1 — APPROVED** [review]

**Pull request merged**